src/Mod/Fem/feminout/importZ88Mesh.py

```diff
@@ -454,6 +454,6 @@ def get_z88_element_type(femmesh, femelement_table=None):
         print('Edge femmesh will be exported as 3D truss element nr 4')
         return 4
     else:
-        print('Neither, edge, face or solid femmesh')
+        print('Neither edge, face, or solid femmesh')
```

Comment:
What I'm interpreting being conveyed here is 
"Neither edge (nor) face (nor) solid femmesh"
So I don't think the comma should be after `Neither`. Correct me if I'm wrong.

Reply:
you are right! should we gone write the nor to be more clear?

Reply:
2 ways to write this:
`Neither edge, face, or solid femmesh`  
or  
`Neither edge nor face nor solid femmesh`